sb-form-server/src/routes/templates.js

```diff
@@ -19,10 +19,15 @@ router.get('/:id', async (req, res) => {
   });
 });
 
-router.post('/:id/submission', (req, res) => {
+router.post('/:id/submission', async (req, res) => {
   const { data } = req.body;
-  email(req.params.id, data);
-  res.status(200).send('ok');
+  try {
+    const result = await email(req.params.id, data);
+    res.status(200).send('ok');
+  } catch (err) {
+    console.error('Error sending email', err);
+    res.status(400).send('Submission has failed, please try again or contact support');
```

Comment:
Get approved text via Dec - there maybe an example in the bot of a phone number we can use already